app/ts/ManualInput/ManualInputTable/CriteriaRows/CriterionRow/DataSourceRow/InputCell/DistributionCellDialog/DistributionInputFields/DistributionInputFields.tsx

```diff
@@ -35,7 +36,7 @@ export default function DistributionInputFields({
   }
 
   function handleKeyDown(event: KeyboardEvent<HTMLDivElement>): void {
-    if (event.keyCode === 13 && !isInputInvalid()) {
+    if (event.keyCode === keycode('enter') && !isInputInvalid()) {
```

Comment:
Excellent. Did you check the rest of the code as well for stray instances by any chance?